lib/zstd.h

```diff
@@ -429,16 +432,15 @@ typedef enum {
      ZSTD_c_experimentalParam2=10,
      ZSTD_c_experimentalParam3=1000,
      ZSTD_c_experimentalParam4=1001,
-     ZSTD_c_experimentalParam5=1002,
-     ZSTD_c_experimentalParam6=1003,
-     ZSTD_c_experimentalParam7=1004,
-     ZSTD_c_experimentalParam8=1005,
-     ZSTD_c_experimentalParam9=1006,
-     ZSTD_c_experimentalParam10=1007,
-     ZSTD_c_experimentalParam11=1008,
-     ZSTD_c_experimentalParam12=1009,
-     ZSTD_c_experimentalParam13=1010,
-     ZSTD_c_experimentalParam14=1011
+     ZSTD_c_experimentalParam5=1003,
```

Comment:
Changing the `enum` nb of a parameter _might_ lead to some downstream issues.

Some binders in other languages, such as Java,
may not access the `enum` definition, and use the known integer value directly.
In which case, changing the meaning of this integer can lead to downstream surprises.

Of course, this is a very niche scenario, so I don't anticipate a lot of real-life issues,
but we also have nothing to gain from introducing this disruption.

So I would keep the other numbers "as is", and accept holes in the number scheme.

Reply:
It was more just a visual decision, so that the whole range of `ZSTD_c_experimentalParam[1-n]` is represented. But, I can see why it might just be preferable to delete it, since it shouldn't really matter to anyone `experimentalParam*` wouldn't be consecutive.